pubber/src/main/java/daq/pubber/PubberSystemManager.java

```diff
@@ -116,7 +112,9 @@ public Date getDeviceStartTime() {
 
   @Override
   public void periodicUpdate() {
-    sendSystemEvent();
+    if (host instanceof Pubber) {
```

Comment:
why this check?

Reply:
It's still sending system events for the gateway and direct devices.